tests/pageObjects/LoginPage.js

```diff
@@ -0,0 +1,22 @@
+class LoginPage {
+    constructor() {
+        this.loginBtnLocator = ".btn.btn-primary.btn-block"; 
+        this.emailFieldLocator = "#spree_user_email";
+        this.passwordFieldLocator = "#spree_user_password";
```

Comment:
Use the term `selector`
```suggestion
        this.passwordFieldSelector = "#spree_user_password";
```